### ISSUE TITLE
Tray icon is not visible on KDE 5.8

### ISSUE THREAD

**Issue by dantalian-pv** [mattermost/desktop#568]
I confirm (by marking "x" in the [ ] below): 

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

**Summary**
There is no tray icon in KDE 5.8 (plasma5-desktop 5.8.7.1).
I've just updated my Linux from OpenSUSE Leap 42.2 to 42.3 with default KDE installation. On the previous SUSE version it worked, but now it is not. I even deleted ~/.config/Mattermost folder and made all settings again, but it didn't help me.

**Steps to reproduce**
- Operating System = Linux OpenSUSE Leap 42.3 with latest updates and plasma5-desktop 5.8.7.1
- Mattermost Desktop App version (3.7.0) 
- Mattermost Server version (Version: 4.0.0  (4.0.1)) 
- Clear steps to reproduce the issue:
1. Run desktop application in plasma5-desktop 5.8 (KDE 5) and select checkboxes in application setting:
* Show icon in the notification area
* Icon theme: Dark
* Leave app running in notification area when application window is closed
2. Restart the app

**Expected behavior**
Tray icon in tray bar.

**Observed behavior**
No tray icon (even after OS restrat).

**Comment by yuya-oc** [comment]
Not reproduced for me. I can see the tray icon on default installation of OpenSUSE Leap 42.3 with KDE Plasma 5.8.7. Would you make sure whether `~/.config/Mattermost/config.json` has `"showTrayIcon": true`?

![screenshot_20170803_222832](https://user-images.githubusercontent.com/1412057/28924249-080d6212-789c-11e7-838d-06a0c5ebb54b.png)

**Comment by dantalian-pv** [comment]
My config.json is:
`
  "teams": [
    {
    }
  ],
  "showTrayIcon": true,
  "trayIconTheme": "dark",
  "minimizeToTray": true,
  "version": 1,
  "notifications": {
    "flashWindow": 2
  },
  "showUnreadBadge": true,
  "useSpellChecker": true,
  "spellCheckerLocale": "en-US",
  "lastMattermostVersion": "3.7.0"
}`

Furthermore, I should see registered tray icon, if I open settings for the tray bar:
![spectacle v17433](https://user-images.githubusercontent.com/5945875/28925098-26da7078-7864-11e7-92c6-25805c034430.png)

**Comment by dantalian-pv** [comment]
I can provide more information (logs, system settings, etc), just tell me how.

**Comment by yuya-oc** [comment]
It's weird. Actually the tray icon is registered to the list on my machine. Unfortunately it's the first time that I use both of OpenSUSE and KDE, so I'm not sure information which I should know.

1. If you are executing the app from command line, possibly the console might show exceptions about tray icon.
2. Please try artifacts of the latest master branch. These use a newer version of Electron, the framework we are using.
    - x64: https://984-43964563-gh.circle-artifacts.com/0/tmp/circle-artifacts.WzhSZ0y/mattermost-desktop-3.7.0-linux-x64.tar.gz
    - ia32: https://984-43964563-gh.circle-artifacts.com/0/tmp/circle-artifacts.WzhSZ0y/mattermost-desktop-3.7.0-linux-ia32.tar.gz

**Comment by dantalian-pv** [comment]
I've fixed my problem by installing libappindicator-gtk3. I've just noticed that the app is based on gtk (Electron) and this was the key for me.
The tray icon looks ugly, and left click does not work, but at least I can see it in my tray bar.
![spectacle s11054](https://user-images.githubusercontent.com/5945875/28957041-695d27c4-78f0-11e7-86dd-fbf67d412d27.png)

Anyway, @yuya-oc thank you for you help. I think we can count this issue as fixed.
